ui/src/components/select/QSelect.js

```diff
@@ -919,6 +919,8 @@ export default Vue.extend({
         },
         click: e => {
           // label from QField will propagate click on the input (except IE)
+          if (e.target.classList.contains('no-outline')) return
```

Comment:
@pdanpdan 

Hi mate,

I have tried your changes and it works exactly the same. Just wondering why do we check `this.useInput !== true`?

on IE `e.target.classList.contains('no-outline')` only captures the click event when you don't use `use-input` prop. It would never reach `e.target.classList.contains('no-outline')` when you use `use-input` so in my opinion the condition here is unnecessary.

However, since your change works perfectly fine so it's up to your preference. It's great that we can solve this issue anyway.

Cheers!